tests/e2e/hellodapr_test.go

```diff
@@ -104,3 +104,16 @@ func TestHelloDapr(t *testing.T) {
 		})
 	}
 }
+
+func TestScaleReplicas(t *testing.T) {
```

Comment:
I wonder if we should split the test in a separate file per scenario or per app.

Reply:
Any thoughts on this?